### PR TITLE
Imix kernel test cleanup for 1.3 release

### PR DESCRIPTION
**Pull request by phil-levis**
### Pull Request Overview

This pull request cleans up the imix kernel tests; it includes all of them in the boot sequence (commented out) for easy inclusion and adds comments to each of them explaining how to run them and the expected output.

### Testing Strategy

This pull request was tested by running all of the tests successfully.

### TODO or Help Wanted

Nothing.

### Documentation Updated

- [X] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [X] Ran `make formatall`.

### CODE REVIEW

File: boards/imix/src/main.rs [hunk 1]
```diff
@@ -117,7 +117,7 @@ const PAN_ID: u16 = 0xABCD;
 const FAULT_RESPONSE: kernel::procs::FaultResponse = kernel::procs::FaultResponse::Panic;
 
 #[link_section = ".app_memory"]
-static mut APP_MEMORY: [u8; 16384] = [0; 16384];
+static mut APP_MEMORY: [u8; 32768] = [0; 32768];
```
Comment (phil-levis) [inline boards/imix/src/main.rs]: Note: this is only peripherally related but didn't want to do a separate PR. It was increased so that imix can run some of the userspace tests which are designed for Hail, which allocates more RAM to apps.

File: boards/imix/src/aes_ccm_test.rs [hunk 2]
```diff
@@ -1,3 +1,16 @@
+//! To run this test, include the code ```
```
Comment (ppannuto) [inline boards/imix/src/aes_ccm_test.rs]: Minor thing: Fenced code blocks need to start on their own line (not sure if this is a general markdown rule or a rust+markdown rule), but currently this looks like:

image

https://deploy-preview-1219--docs-tockosorg.netlify.com/imix/aes_ccm_test/index.html

Comment (phil-levis) [reply boards/imix/src/aes_ccm_test.rs]: Fixed.
